helmfile/values/ingress-nginx.yaml.gotmpl

```diff
@@ -1,12 +1,17 @@
 ## nginx configuration
-## Ref: https://github.com/kubernetes/ingress/blob/master/controllers/nginx/configuration.md
+## Ref: https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/nginx-configuration/index.md
 ##
+
 controller:
+  name: controller
```

Comment:
```suggestion
```
This can be removed, following Fredriks suggestion about slimming down the values.

Reply:
done